solr/CHANGES.txt

```diff
@@ -124,6 +124,9 @@ Improvements
 
 * SOLR-17063: Do not retain log param references in LogWatcher (Michael Gibney)
 
+* SOLR-16397: The command-status v2 endpoint has been updated to be more REST-ful.
```

Comment:
[0] Solr has a similar API for requesting the status of collection level operations, so it might be worth clarifying here which one was updated.